x-pack/plugins/monitoring/server/lib/elasticsearch/get_last_recovery.ts

```diff
@@ -78,9 +86,7 @@ export function handleMbLastRecoveries(resp: ElasticsearchResponse, start: numbe
     (hit) => hit._source.elasticsearch?.index?.recovery
   );
   const filtered = mapped.filter(filterOldShardActivity(moment.utc(start).valueOf()));
-  filtered.sort((a, b) =>
-    a && b ? (b.start_time_in_millis ?? 0) - (a.start_time_in_millis ?? 0) : 0
-  );
+  filtered.sort((a, b) => (a && b ? (b.start_time.ms ?? 0) - (a.start_time.ms ?? 0) : 0));
```

Comment:
I don't know if we can assume this property to exist on all metricbeat versions. I'd be inclined to be defensive here as well